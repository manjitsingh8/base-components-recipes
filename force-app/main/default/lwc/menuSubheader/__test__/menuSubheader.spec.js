/*
 * Copyright (c) 2019, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */

import { createElement } from 'lwc';
import Element from 'c/menuSubheader';

const createMenuSubheader = () => {
    const element = createElement('c-menu-subheader', { is: Element });
    document.body.appendChild(element);
    return element;
};

describe('c-menu-subheader', () => {
    it('default state', () => {
        const menuSubheader = createMenuSubheader();
        menuSubheader.label = 'Subheader';

        return Promise.resolve().then(() => {
            expect(menuSubheader).toMatchSnapshot();
        });
    });
});
