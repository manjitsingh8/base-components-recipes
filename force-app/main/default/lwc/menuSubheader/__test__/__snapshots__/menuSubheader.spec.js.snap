// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`c-menu-subheader default state 1`] = `
<c-menu-subheader
  class="slds-dropdown__header slds-truncate"
  role="separator"
>
  #shadow-root(open)
    <span>
      Subheader
    </span>
</c-menu-subheader>
`;
